pymatgen/core/ion.py

```diff
@@ -19,15 +23,15 @@ class Ion(Composition, MSONable, Stringify):
     Mn[+2]. Note the order of the sign and magnitude in each representation.
     """
 
-    def __init__(self, composition, charge=0.0, _properties=None) -> None:
+    def __init__(self, composition: Composition, charge: float=0.0, _properties=None) -> None:
```

Comment:
`_properties` is not used. Any good reason to keep it here?

Reply:
not sure. maybe for `monty` `as/from_dict` compatibility? but since there's no comment explaining it, i'm open to removing it to see what happens

Reply:
Removed in 11a53c4b26a39bc686101dc888b0f680d9e36ab1. I would keep an eye this as well.